Day 2, Q1 and Q2 Solutions.

sanjaypradeep opened the pull request:
Forked form PyLenin - Q1 solved.

@pylenin please review

Commit:
Day 2, Q1 Solution.

Forked form PyLenin - Q1 solved.

Commit:
Day 2, Q2 solutions.

Day 2, Q2 solutions.. good understanding about date difference via timedelta type.

pylenin commented on .gitignore:
```diff
@@ -0,0 +1,7 @@
+
```
This file is not required. Remove it.

sanjaypradeep replied:
It may be required, because in future, this will help people whoever clones to their local and do changes. isn't it?

pylenin replied:
Ideally, I should be adding this to the repo. Didn't expect these `.idea` files. Its okay. I will accept it. Thanks for improving the repo.

pylenin commented on Day 2 - Python Datetime objects and Timedelta/Q2_TimeDifference.py:
```diff
@@ -0,0 +1,15 @@
+
+time_difference = future_date - old_date
+
+print("Difference in days.. ", (time_difference.days*24 + time_difference.seconds/3600)/24)
```
Your approach is correct. But your answer is providing a decimal number. 
In the question, it asks to provide the time difference. Then it says mention both the days and time. Which means, you should provide your answer like "Time difference is x days y hours z mins...so on". Is it clear?

sanjaypradeep replied:
Comments addressed!

pylenin replied:
LGTM !!

Commit:
Addressing comments.

@pylenin, addressed the comments, expected output is obtained. seconds component is not included as of now, let me know if needed will update that as well.
